content/en/docs/Concepts/MVC.md

```diff
@@ -8,11 +8,127 @@ description: >
+    );
+});
+
+```
+
+This code also shows how endpoints can be used to determine what routing should be used.
+
+## Areas
+
+Areas are an organizational tool in MVC that allows programs to have much more effective routing and a set hierarchy to operate with. Areas are effectively a structure; thus, their content corresponds to a specific function or purpose in a program. For example, a program designed for  car dealership may have areas like "Rent," "Purchase," and "Sell," in order to reasonably separate the code. Areas are important in large programs in order to separate code in functional categories that may differ from simply their form (instead of grouping all controllers or all models, group their functions together).
+	
+In terms of routing, areas can simplify the routes necessary for certain pages and also group similar pages together. For example, examine the following code:
+
+```csharp
+app.UseEndpoints(endpoints =>
+{
+    endpoints.MapControllerRoute(
+        name: "Areas",
+        pattern: "{area:exists}/{controller=Home}/{action=Index}/{id?}");
+
+    endpoints.MapControllerRoute(
+        name: "default",
+        pattern: "{controller=Home}/{action=Index}/{id?}");
+});
+```
+
+Now, all "Purchase" or "Sell" pages can be accessed directly from referencing their area in the URL. Without this routing, these pages could exist in all different routes based on controllers they interact with.
+
+## Action Methods
+
+Action methods are defined in controllers to associate user actions with a program output. Actions are associated with an "ActionResult," which includes any of the following:
+
+| Action Methods |
+| -------- |
+| ViewResult |
+| EmptyResult |
+| ContentResult |
+| FileContentResult |
+| FilePathResult |
+| FileStreamResult |
+| JavaScriptResult |
+| JsonResult |
+| RedirectResult |
+| RedirectToRouteResult |
+| PartialViewResult |
+| HttpUnauthorizedResult |
+
+Default actions are defined in each controller. Action methods can also include parameters of any type if necessary. Here is an example of defining an Action Method:
+
+```csharp
+public ActionResult Index()
+{
+    return View(); 
+}
+```
+
+Now, when /Index is referenced in the URL, the program will return the corresponding view.
+
+## Request Types
+
+Request Types identify the different possible actions that are invoked upon a program, often in order to retrieve and/or send data. Requests that aren't "GET" are defined by writing Http followed by the request type (as seen below) before an action method:
+
+```csharp
+public class StudentController : Controller 
+
+[HttpPost] 
+
+public ActionResult PostAction() // handles POST requests by default
+{ 
+    return View("Index"); 
+}
+```
+
+Examples of Request Types include:
+* HttpGet: Retrieve information identified by a request
+* HttpPost: Annotate, post messages, submitting a form, etc.
+* HttpPut: Send data to be stored
+* HttpHead: Same as HttpGet without a message-body
```

Comment:
Remove all but the GET, POST, PUT, and DELETE methods. These are the 4 core ones that are necessary.